src/app/App.jsx

```diff
@@ -21,11 +15,17 @@ function App() {
     <Wrapper>
       <AppHeader user={user} handleLogoutStatus={setUser} />
```

Comment:
AppHeader 에서 onLogout prop을 받고있는데 App에서는 안 보내주고 잇네요.    
아래와 같이 넘겨줘야 될 것 같네요
```suggestion
      <AppHeader user={user} handleLogoutStatus={setUser} onLogout={onLogout} />
```